client/cypress/integration/app.spec.ts

```diff
@@ -9,24 +9,10 @@ describe('App', () => {
     page.getAppTitle().should('contain', 'Handy Pantry');
   });
 
-  it('The sidenav should open, navigate to "Products" and back to "Pantry"', () => {
-    // Before clicking on the button, the sidenav should be hidden
-    page.getSidenav()
-      .should('be.hidden');
-    page.getSidenavButton()
-      .should('be.visible');
-
-    page.getSidenavButton().click();
+  it('Should have a working toolbar', () =>{
```

Comment:
It's probably a good idea to include a check that the navbar exists before trying to test the navigation. Something like cy.get(<toolbar-indicator>).should('have.text', 'My Pantry') or something similar. This could potentially save some hassle if it were to break in the future.

Reply:
Good idea, added.